plugins/GeoIp2/GeoIP2AutoUpdater.php

```diff
@@ -741,4 +744,28 @@ public static function isDbIpUrl($url)
     {
         return !! preg_match('/db-ip\.com/', $url);
     }
+
+    private function isPaidDbIpUrl($url)
+    {
+        return !! preg_match('/db-ip\.com\/account\/[0-9a-z]+\/db/', $url);
+    }
+
+    private function fetchPaidDbIpUrl($url)
+    {
+        $content = Http::fetchRemoteFile($url);
```

Comment:
might be good to trim `$content` just to be safe

Reply:
```suggestion
        $content = trim(Http::fetchRemoteFile($url));
```